minesweeper.py

```diff
@@ -40,6 +40,21 @@ def mines_around_square(self, coordinate):
                 adjacent_mines += 1
         return adjacent_mines
 
+    def win_animation(self):
+        gif_label.place(relx=0.5, rely=0.5, anchor="center")
+
+        def update(index):
+            frame = frames[index]
+            index += 1
+            if index >= 8:
```

Comment:
```suggestion
            if index >= len(frames):
```

Repeating `8` is now worse than before because the two `8`'s are far apart.